editions/tw5.com/tiddlers/gettingstarted/GettingStarted.tid

```diff
@@ -1,12 +1,12 @@
 created: 20131129090249275
-modified: 20200507203622933
+modified: 20211116234627990
 tags: [[Working with TiddlyWiki]]
 title: GettingStarted
 type: text/vnd.tiddlywiki
 
-Click here to download an empty copy of TiddlyWiki: {{$:/editions/tw5.com/snippets/download-empty-button}}
+Out of the box, TiddlyWiki is an HTML file. Click here to download an empty HTML copy of TiddlyWiki: {{$:/editions/tw5.com/snippets/download-empty-button}}
```

Comment:
I'm not sure "Out of the box" is helpful here. It doesn't convey that this is one of multiple forms that TiddlyWiki can take.

Reply:
I'm just trying to convey, without interrupting the flow, and in as few words as possible, that TiddlyWiki is different from the applications that people have come to understand and install. The point is to eliminate the confusion when what they download isn't' an exe or  msi or anything they're familiar with.

Reply:
Yes indeed, the intent is very clear, I just worry that the wording is too clumsy.

How about a "What is this?" link under the button that links to a tiddler explaining what's going on  (hopefully there'll be an existing one).